src/components/toast/toast.d.ts

```diff
@@ -4,25 +4,9 @@ export interface ToastPropTypes {
   /** The rendered children of the component. */
   children: React.ReactNode;
   /** Customizes the appearance in the DLS theme */
-  variant?:
-    | "default"
-    | "error"
-    | "help"
-    | "info"
-    | "maintenance"
-    | "new"
-    | "success"
-    | "warning";
+  variant?: "error" | "info" | "success" | "warning";
```

Comment:
Let's tidy this up a bit with a `type ToastVariants = "error" | "info" | "success" | "warning";` which can be shared on both these props here